cypress/tests/integration/growers/[growerid].cy.js

```diff
@@ -0,0 +1,25 @@
+import grower from '../../../../doc/examples/growers/100.json';
+import { prepareNocks, clearNocks } from '../nockRoutes';
+
+beforeEach(() => {
+  clearNocks();
+});
+
+describe('Grower Page', () => {
+  it('renders with grower data', () => {
+    const path = `/growers/${grower.id}`;
+    cy.fixture('images/grower.png').then((image) => {
+      const blob = Cypress.Blob.base64StringToBlob(image, 'image/png');
+      const image_url = Cypress.Blob.createObjectURL(blob);
+      prepareNocks({ grower: { ...grower, image_url } });
+    });
+
+    cy.visit(path, {
+      failOnStatusCode: false,
+    });
+
+    cy.url().should('include', '/growers');
+    cy.contains(grower.first_name);
```

Comment:
@sam-rice we are improving all the tests by adding strict and complete checking the elements on pages as many as possible, so can you add all those elements/content of the grower, like last name and other things.